prac_05/state_names.py

```diff
@@ -7,6 +7,7 @@
 CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
```

Comment:
Task asks for the states to be printed neatly lined up using a loop.

Reply:
Loop is located on lines 11 & 12.